examples/config/grid_config.config

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+
+<plugin filename="MinimalScene">
+    <ignition-gui>
+      <title>View 1</title>
+      <property type="string" key="state">docked</property>
+    </ignition-gui>
+    <engine>ogre2</engine>
+    <scene>scene</scene>
+    <ambient_light>1 1 1</ambient_light>
+    <background_color>0.8 0.8 0.8</background_color>
+    <camera_pose>-6 3 6 0 0.5 0</camera_pose>
+</plugin>
+<plugin filename="InteractiveViewControl" name="Interactive view control">
+  <ignition-gui>
+    <anchors target="View 1">
+      <line own="right" target="right"/>
+      <line own="top" target="top"/>
+    </anchors>
+    <property key="resizable" type="bool">false</property>
+    <property key="width" type="double">5</property>
+    <property key="height" type="double">5</property>
+    <property key="state" type="string">floating</property>
+    <property key="showTitleBar" type="bool">false</property>
+  </ignition-gui>
+</plugin>
+<plugin filename="GridConfig" name="Grid config">
+  <ignition-gui>
+    <property type="string" key="state">docked</property>
+  </ignition-gui>
+  <cell_count>3</cell_count>
+  <vertical_cell_count>2</vertical_cell_count>
+  <cell_length>2</cell_length>
+  <pose>1 2 3  0 0 0</pose>
```

Comment:
```suggestion
  <pose>1 2 3 0 0 0</pose>
```

Reply:
I've been trying to get in the habit of separating position and rotation with an extra space as suggested on [Specifying pose: Proposal for a better pose](http://sdformat.org/tutorials?tut=better_pose_proposal&cat=pose_semantics_docs&#motivation):

_libsdformat and SDFormat tutorials should encourage additional whitespace, e.g. to separate translation and rotation, use 3 spaces (instead of 1) as a delimiter between values if they fit on one line, or use a newline (possibly with hanging indents) if they do not fit on one line._